### ISSUE TITLE
get_exercise: invalid literal for int() with base 10: ''

### ISSUE THREAD

**Issue by rbelzile** [coddingtonbear/python-myfitnesspal#52]
When I attempt to use the get_exercise code, I get the following error.

```
>>> import myfitnesspal
>>> client = myfitnesspal.Client('********', '********')
>>> client.get_exercise(2017,9,1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "myfitnesspal\client.py", line 432, in get_exercise
    exercise = self._get_exercise(document)
  File "myfitnesspal\client.py", line 381, in _get_exercise
    value = self._get_numeric(column.text)
  File "myfitnesspal\client.py", line 220, in _get_numeric
    return int(re.sub(r'[^\d.]+', '', string))
ValueError: invalid literal for int() with base 10: ''
```

I believe it's related to the fact that my iPhone syncs my step data in and returns N/A for minutes, which means that the re.sub... in _get_numeric is returning an empty string to int()

I can fix this, but I already have a pull request open and I'm unsure semantically if it makes more sense to check in _get_numeric and return 0 for blank strings or whether the get_exercise should be parsing for entirely non-numeric content before calling _get_numeric

**Comment by jgissend10** [comment]
That's exactly what it's related to. I have the same issue.

**Comment by jgissend10** [comment]
In addition, when fixing this so that if 'N/A' in column.text store as None, I discovered that Fitbit calorie adjustments show up with a blank name.
They have two a tags in the td.

`<td class="first alt">

              <div class="exercise-description">

                  <a onclick="showAdjustmentInfo(450,550,2867241370,'en' ); return false;" href="#">
                    
                    
                    Fitbit calorie adjustment
                  </a>

              </div>


                <a onclick="showAdjustmentInfo(450,550,2867241370,'en' );" style="cursor: pointer;">
                  <img alt="Ic_i" src="http://d34yn14tavczy0.cloudfront.net/assets/ic_i-5a18e414bbf2a80fbfa2bed9c28c79c7.png">
                </a>

            </td>`

The if statement for grabbing the name returns the non nested a tag which is empty.

`if columns[0].find('a') is None or not columns[0].find('a').text.strip():`

^ That seems to fix it. I'll try to put in a PR soon with this fix.

**Comment by jgissend10** [comment]
#55 This is the related PR

**Comment by 1minus1** [comment]
I am having this same issue. Any chance for a fix?

**Comment by jgissend10** [comment]
I make a PR and it got merged. But yesterday I had the same issue again. I installed from source instead of pip install MyFitnessPal and it worked. Try doing a pip install git+https://GitHub.com/Coddingtonbear/Python-myfitnesspal.git

**Comment by 1minus1** [comment]
Done, that fixed it. Thanks.

**Comment by jgissend10** [comment]
No problem. I guess the pip version is stale?

**Comment by coddingtonbear** [comment]
Ahh; hrm -- sorry about that; I'll push out a new release later today!

**Comment by coddingtonbear** [comment]
Released @jgissend10's fix as part of 1.11.0; sorry for the delay!
